pom.xml

```diff
@@ -378,6 +378,7 @@
             <configuration>
               <executable>${jvm.executable}</executable>
               <arguments>
+                <argument>-Des.path.home=.</argument>
```

Comment:
This exec plugin won't ensure the build directory even exists! Hell, it doesn't even ensure the code is compiled, or give you any output from elasticsearch at all. I don't know why people use it, but ES expects a directory structure here, you know with conf/ and what not. So we'd have to actually prepopulate this eshome and so on.

Basically, I don't think we should try to improve on this plugins usage at all. IMO this plugin sucks for doing this, and instead if this build target is really needed, we should use antrun-plugin+profile to do it.

But this target has likely been broken for some time, and nobody seems to have noticed.